packages/wrangler/src/api/startDevWorker/events.ts

```diff
@@ -149,4 +150,5 @@ export type ProxyData = {
 	liveReload?: boolean;
 	proxyLogsToController?: boolean;
 	internalDurableObjects?: CfDurableObject[];
+	entrypointAddresses: WorkerEntrypointsDefinition | undefined;
```

Comment:
Noticed you've added this property and set it everywhere it's needed but I don't see where it's read/used